### PR TITLE
fix: profile photo

### PR DESCRIPTION
**Pull request by amitojsingh**

### CODE REVIEW

File: src/pages/profile/index.tsx [hunk 1]
```diff
@@ -88,56 +90,46 @@ export default function Profile() {
     }
   };
 
+  async function updateUserAndProfile(usr: User, displayName: string, pURL: string, uname: string) {
+    const updatedUserData = {
+      ...usr,
+      displayName,
+      pURL,
+      uname: uname !== usr.username ? uname : usr.username,
```
Comment (MySmilingTurban) [inline src/pages/profile/index.tsx:98]: should be named as photoURL and username instead of pURL and uname

File: src/pages/profile/index.tsx [hunk 2]
```diff
@@ -88,56 +90,46 @@ export default function Profile() {
...
+  }
+
   const handleSubmit = async () => {
     if (user && usernameError === '') {
       // check if anything has changed
-      if (
+      const isUnchanged =
         name === user.displayName &&
         username === user.username &&
         photoURL === user.user?.photoURL &&
```
Comment (MySmilingTurban) [inline src/pages/profile/index.tsx:115]: should we also add a check with photoURL === user.photoURL?

File: src/pages/profile/index.tsx [hunk 3]
```diff
@@ -225,6 +217,7 @@ export default function Profile() {
 
   useEffect(() => {
     const userDetails = user.user;
+    console.log(user);
```
Comment (MySmilingTurban) [inline src/pages/profile/index.tsx:220]: nit: remove console statement

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
minor changes

File: src/pages/profile/index.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/pages/profile/index.tsx:115]: user.photoURL? or user.user?.photoURL
photoURL === user.user?.photoURL
this is already there

File: src/pages/profile/index.tsx (hunk 3 above)
Comment (amitojsingh) [reply src/pages/profile/index.tsx]: fixed

File: src/pages/profile/index.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/pages/profile/index.tsx]: fixed

File: src/pages/profile/index.tsx (hunk 2 above)
Comment (MySmilingTurban) [reply src/pages/profile/index.tsx:116]: we can have both as condition to make sure they are in sync
user.photoURL is coming from our users collection from firebase
whereas user.user.photoURL comes from firebase auth

File: src/pages/profile/index.tsx [hunk 4]
```diff
@@ -13,6 +13,8 @@ import { showToastMessage } from 'utils';
 import { uploadImage } from 'utils/storage';
 import CONSTANTS from 'constants/constant';
 import { User } from 'types';
+import { Timestamp } from 'firebase/firestore';
+
```
Comment (MySmilingTurban) [inline src/pages/profile/index.tsx:17]: nit: remove extra line

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
one minor fix

File: src/pages/profile/index.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/pages/profile/index.tsx:116]: okay.

Comment (amitojsingh) [reply src/pages/profile/index.tsx:116]: fixed veer ji

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**